src/modules/layout/components/__tests__/LocationHeader.spec.js

```diff
@@ -96,7 +105,10 @@ describe('LocationHeader', () => {
     const component = shallow(<LocationHeader matchRoute={matchRoute} language={language}
                                               locationModel={createLocation(true, true)}
                                               currentPath=''
-                                              viewportSmall />)
+                                              viewportSmall
+                                              eventCount={0} />)
     expect(component.dive()).toMatchSnapshot()
   })
+
+  // fixme: Test the events enabled functionality. Especially isEventsActive()
```

Comment:
@NKlug eventCount was missing in this test. Some more tests are needed here to keep the coverage up :)

Reply:
I'll have a look at it :)